frontend/src/pages/Workspace.tsx

```diff
@@ -0,0 +1,20 @@
+// import { Routes, Route, Navigate } from 'react-router-dom';
+import { Outlet } from 'react-router-dom';
+import { CloudIcon, BookOpenIcon } from '@heroicons/react/24/solid'
+
+export default function Workspace() {
```

Comment:
Currently the `/` path just shows you a blank page with the header. Should we redirect `/` to `/home` automatically?